src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -124,61 +129,95 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
 
     def tearDown(self):
+        """Tear down after each test case."""
         super(TestWithoutServers, self).tearDown()
 
         if self.fault_file:
             os.remove(self.fault_file)
 
+
 class TestWithServers(TestWithoutServers):
-    '''
-    Run tests with DAOS servers
+    """Run tests with DAOS servers and at least one client.
+
+    Optionally run DAOS clients on specified hosts.  By default run a single
+    DAOS client on the host executing the test.
 
     :avocado: recursive
-    '''
+    """
+
     def __init__(self, *args, **kwargs):
+        """Initialize a TestWithServers object."""
         super(TestWithServers, self).__init__(*args, **kwargs)
 
         self.agent_sessions = None
 
     def setUp(self):
+        """Set up each test case."""
         super(TestWithServers, self).setUp()
 
-        self.server_group = self.params.get("server_group", '/server/',
-                                            'daos_server')
-        self.hostlist_servers = self.params.get("test_machines", '/run/hosts/*')
-
+        self.server_group = self.params.get(
+            "server_group", "/server/", "daos_server")
```

Comment:
Should this be looking for "name" in "server_config"?   I do not see any matches in any yaml file for  "server_group" currently.

Reply:
Yes, it seems so.

Reply:
The change has been included in latest commit